021-special-repository-for-st-clients-based-on-example-clients.md

```diff
@@ -0,0 +1,61 @@
+# Special repository for ST clients based on example clients
+
+This proposal suggests creating a new repository for `Strimzi` ST client based on 
+[Strimzi client-examples](https://github.com/strimzi/client-examples).
+
+## Current situation
+
+Currently, we are using two clients in our STs:
+ - `InternalKafkaClients` (based on `test-client` image) 
```

Comment:
I think you could describe briefly `test-client` and why it's not sufficient/you want to replace it